tests/php/Authenticator/LoginHandlerTest.php

```diff
@@ -232,24 +233,77 @@ public function cannotSkipMFAProvider()
         ];
     }
 
-    public function testSkipRegistration()
+    /**
+     * @param $memberFixture
+     * @param $expectedRedirect
+     * @dataProvider skipRegistrationProvider
+     */
+    public function testSkipRegistration($memberFixture, $mfaRequiredInGrace = false, $expectedRedirect = null)
     {
-        $this->setSiteConfig(['MFARequired' => false]);
+        if ($mfaRequiredInGrace) {
+            $this->setSiteConfig([
+                'MFARequired' => true,
+                'MFAGracePeriodExpires' => DBDatetime::now()
+                    ->setValue(strtotime('+1 day', DBDatetime::now()->getTimestamp()))->getTimestamp()
```

Comment:
I think we prefer using `->Rfc2822()` in place of `->getTimestamp()` when setting DBDateTime values?

Reply:
Thanks